### DISCUSSION TITLE
Upgrade 1.30.11 to 2.0.27 - login possible, css-styles missing

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4163] [Questions & Answers] [answered]
Any Idea what happend here? 
I followed the steps described in the [documentation](https://www.kimai.org/documentation/updates.html), (and also executed the [helper-script](https://www.kimai.org/documentation/cache.html#helper-script)). 

Login-Screen was possible, but the Login-Screen and the following wizard (you see after login) do miss some stylesheets.  
![image](https://github.com/kimai/kimai/assets/1778068/3ab34580-179e-4947-a6bf-de9ca1a2cdbf)

What could be the reason for this?

There is a directory called bundles in `public` of the project, but there is no directory `tabbler` inside: 

![image](https://github.com/kimai/kimai/assets/1778068/2d7fff63-85a0-4216-a71c-0188564175c4)

![image](https://github.com/kimai/kimai/assets/1778068/6279ad65-8a87-419b-8702-93e25902d191)

I didn't had installed any plugin in my old 1.30.11, nor I'd install any after the update. 
Any suggestions on this?

**Comment by kevinpapst** [comment, marked as answer]
You have somehow managed to get a half updated system.
I assume you ignored some errors during the upgrade or composer run.
`git status` should be clean.

Run `composer install ... ` according to the docs again.

**Reply by xstable** [reply]
Wow, great, that does the job. Thanks a lot kevin. 
Kimai 2.0 looks awesome.
